src/index.tsx

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
 import './styles/index.scss';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import { store } from './store';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const Root = () => (
+  <Provider store={store}>
```

Comment:
How about keeping the index.tsx intact and do this in App component instead?

Reply:
Done